packages/apollo-server-express/src/__tests__/ApolloServer.test.ts

```diff
@@ -426,13 +427,13 @@ describe('apollo-server-express', () => {
         });
       });
     });
-    describe('file uploads', () => {
+    // NODE: Intentionally skip file upload tests on Node.js 10 or higher.
+    (
+      atLeastMajorNodeVersion(10)
+      ? describe.skip
+      : describe
+    )('file uploads', () => {
       it('enabled uploads', async () => {
-        // XXX This is currently a failing test for node 10
-        const NODE_VERSION = process.version.split('.');
-        const NODE_MAJOR_VERSION = parseInt(NODE_VERSION[0].replace(/^v/, ''));
-        if (NODE_MAJOR_VERSION === 10) return;
```

Comment:
This technique of `return`ing fails to deliver that intentional "skip" to Jest's reporter, so it's best to avoid it.